tools/tune/crossover/crossover_gen_coefs.m

```diff
@@ -54,7 +54,7 @@
 		b = [1 - cutoff, 0, 0];
 		a = [1, 0, 0];
 		return;
-	endif
```

Comment:
Extra semicolon is superfluous at line number `1`, `13`, `18`, `25` and `gain_db` as input argument might be unused , please consider removing or replacing with `~`

Reply:
Yep, good to remove the extra semicolons. I'll leave for now the gain_db in case there's plan to add support for it later. Could multiply lp.b with 10^(gain_db/20) but I'll leave to later since it would need additional checking. Is it OK @thesofproject/google ?